models/account.model.js

```diff
@@ -40,6 +40,14 @@ const AccountSchema = new mongoose.Schema({
         type: String,
         enum: Constants.USER_TYPES,
         default: Constants.HACKER
+    },
+    birthdate: {
+        type: Date,
+        required: true
+    },
+    phoneNumber: {
+        type: Number,
```

Comment:
Perhaps make this a string, with regexp constraints? Idk what the best way to save a phone number is